mysql/position.go

```diff
@@ -6,12 +6,22 @@ import (
 	"strings"
 )
 
-// For binlog filename + position based replication
+// Position for binlog filename + position based replication
 type Position struct {
 	Name string
 	Pos  uint32
 }
 
+// Compare the position information between the p and o,
+// if p > o return 1 means the position of p is further back than o.
+// example is following
+// id | project
+// 1  | math
+// 2  | cs
+// 3  | ee
+// p at id 3 and o at id 1.
+// so this way we can determine from which position we should start syncing the data
+// so that it is not consumed repeatedly.
```

Comment:
```suggestion
// Compare the position information between the p and o,
// if p > o return 1 means the position of p is further back than o.
```

because the example is too simple for binlog replication scenario, I think adding that example will cause more trouble. For example, multiple row changes may have same position, and not all position can (re)start a binlog replication.